drivers/adc-dac/ad74413r/iio_ad74413r.c

```diff
@@ -309,6 +310,8 @@ static int ad74413r_iio_read_offset(void *dev, char *buf, uint32_t len,
 		} else {
 			ad74413r_get_adc_range(iio_desc->ad74413r_desc,
 					       channel->address, &reg_val);
+			if (ret)
+				return ret;
```

Comment:
am I missing something here? I don't see where ret value is set

Reply:
`ret` has to be set, will change.